codes/1963_cpp_yji0828.cpp

```diff
@@ -0,0 +1,64 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <iostream>
+#include <cstdio>
+#include <queue>
+#include <string>
+#include <cstring>
+#include <cmath>
+using namespace std;
+queue<int> q;
+int check[10001];
+bool prime[10001];
+void bfs(int start, int dest);
+void eratos();
+
+
+int main() {
+	int T;
+	int start, dest;
+	eratos();
+	scanf("%d", &T);
+	for (int i = 0; i < T; i++) {
+		memset(check, -1, sizeof(check));
+		scanf("%d %d", &start, &dest);
+		bfs(start, dest);
+		if (check[dest] != -1)
+			printf("%d\n", check[dest]);
```

Comment:
로직을 전역변수를 사용해서 짜기 보다 bfs가 int를 리턴해서 짜는 편이 더 좋습니다.
```suggestion
		ret_bfs = bfs(start, dest);
		if (ret_bfs != -1)
			printf("%d\n", ret_bfs);
```

Reply:
수정했습니다!